Marlin/Configuration.h

```diff
@@ -81,6 +81,20 @@
 // example_configurations/SCARA directory.
 //
 
+//===========================================================================
+//======================== Piston Extruder Printer ==========================
+//===========================================================================
+// For a Piston Extruder printer replace configuration.h with the file in the
+// example_configurations/MMM directory.
+//
+
+//#define PISTON_EXTRUDER //used for the Piston Extruder dryer fan
+//#define E_ADJUST_MANUALLY //variable resistor has one end to ground, other to +5 volts. & wiper to E_ADJUST_PIN
```

Comment:
Yes it is more generally useful. I can take out the ====Piston Extruder==== section and just label the configuration.h variables something like 'extrusion rate adjust'.
What do you think of that change?